containers/cosmos-sdk/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM cgr.dev/chainguard/wolfi-base:latest@sha256:0f1d81605bda6e2388c3c7f731700d8c12e17259d58ffba11f36ddc81d9c0a76
```

Comment:
What's the difference between this and alpine? Why not alpine?

Reply:
I no longer recommend using alpine linux, because of compatibilty issues I had with DNS and glibc absence that may complexify deps management for building and runtime.

this well written blog post will sum it better than me : https://martinheinz.dev/blog/92

I attended a CNCF meeting a few months ago and learned about wolfi-os.
https://thenewstack.io/meet-wolfi-the-linux-distro-designed-to-shrink-your-supply-chain/

TLDR : 
- Better security (less CVEs, SBOMs)
- Lower footprint (cosmos-sdk image is ~35MB with a 25MB binary)
- No compatibility issues

On the cons side, project is less known (but migrating images to another base do not require rewrites) and there are less packages on the manager than alpine, but not very relevant for our use case imho.